File: Rmm.WebClient/src/app/devicePanel-module/models/deviceInfo.ts
import { DeviceStatus } from '../enums/deviceStatus';
import { DeviceStaticInfoResponse } from './response/deviceStaticInfo-response';

export class DeviceStaticInfo {
    public id!: string;
    public name?: string;
    public os?: string;
    public coordinates?: number[];

    constructor(resp: DeviceStaticInfoResponse) {
        this.id = resp.id;
        this.name = resp.name;
        this.os = resp.os;
        this.coordinates = resp.coordinates;
    }
}

File: Rmm.Domain/Rmm.Domain.DAL/Entities/DeviceInfo.cs
using System;
using Rmm.Domain.DAL.Enums;

namespace Rmm.Domain.DAL.Entities
{
    public class DeviceInfo : BaseEntity
    {
        public string Name { get; set; }
        public DeviceStatus Status { get; set; }
        public string Os { get; set; }
        public double[] Coordinates { get; set; }
        public int RunTimeS { get; set; }
        public DateTime LastAnswerTime { get; set; }
    }
}
